### ISSUE TITLE
Update GA event category for a few widget events

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#4006]
Related to our GA tracking event refinements, in addition to the Idea Hub widget-specific events (see #4005), we should also adjust the existing events for All Traffic and PageSpeed accordingly.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The `all_traffic_widget` GA event category should be renamed to `dashboard_all-traffic-widget` across all usages (including widget tooltip tour).
* The `pagespeed_widget` GA event category should be renamed to `dashboard_pagespeed-widget` across all usages (including widget tooltip tour).

## Implementation Brief

* 

### Test Coverage

* 

### Visual Regression Changes

* 

## QA Brief

* 

## Changelog entry

* N/A

**Comment by eugene-manuilov** [comment]
@felixarntz do you mind reviewing IB?

**Comment by felixarntz** [comment]
IB ✅

**Comment by johnPhillips** [comment]
Just a note here that a couple of points were missed during IB / IBR and execution: adding the events to the all traffic widget and pagespeed feature tooltip tours.

I've suggested a way to make the events in feature tours accept the view context dynamically in https://github.com/google/site-kit-wp/issues/4057

I discussed this with @tofumatt and we agreed it might be more straightforward to add these two feature tour events to the ACs for that issue instead of trying to cover it here. Then `viewContext` can be interpolated into the event string nicely. 

CC @tofumatt @eugene-manuilov

**Comment by felixarntz** [comment]
@johnPhillips Maybe for this issue we could hard-code the feature tours to rely on `VIEW_CONTEXT_DASHBOARD` as an intermediate solution? Technically this should be sufficient for now, since those feature tours aren't really shown elsewhere.

**Comment by eugene-manuilov** [comment]
@johnPhillips I agree with Felix. Nothing blocks us from using the `VIEW_CONTEXT_DASHBOARD` constant. Let's hard-code it, for now, to make this ticket complete and then update it in #4057.

**Comment by johnPhillips** [comment]
Good shout - I'll go ahead and use the constant as an intermediary step, and then make them dynamic as part of #4057 👍

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

@eugene-manuilov any ideas what version of SK I will need to be on to trigger the `widget tooltip tour`?  It references it in the AC here. I know what tasks need to be completed to trigger it, just couldn't remember the version. 🤦

**Comment by eugene-manuilov** [comment]
@wpdarren we only have a tooltip tour for the All Traffic widget and it's version is `1.25.0`. There is no widget tooltip tour for the PSI widget although it was mentioned in AC :man_shrugging:.

**Comment by wpdarren** [comment]
### QA Update: ✅ 

The `all_traffic_widget` GA event category is renamed to {`viewContext}_all-traffic-widget` across all usages.
The `pagespeed_widget` GA event category is renamed to `{viewContext}_pagespeed-widget` across all usages.
